Sharpmake/Project.cs

```diff
@@ -1915,7 +1923,8 @@ public class CSharpProject : Project
             ".manifest"
         );
         public Strings ContentExtension = new Strings();
-        public Strings VsctExtension = new Strings(".vsct");
+		public ItemCopyType DefaultContentCopyOperation = ItemCopyType.None;
```

Comment:
There a few places I noticed tabs instead of spaces. We use 4 spaces for indentation in Sharpmake, could you fix the offending lines?

Reply:
Since the solution is still for 2015 the editor config wasn't working by default :( I'll go through the change and clean it up. Is a solution upgrade coming with the new set of changes?

Reply:
Even better! I have a changelist that will make Sharpmake generate its own project. It should be pretty easy to generate VS2017 solutions once this is in. It's going to come with a massive chunk of various other changelists we worked on internally though, so I would integrate your pull request first.

Reply:
Looking forward to it, I'll get mine fixed with the whitespace, rebase when your changes come in, then we can revisit this.

Reply:
As of today, it is not planned to update sharpmake to vs2017.
But, what Benoit is talking about is pretty much already there, if you run UpdateSharpmakeProjects.bat sharpmake will update its csprojs and sln, using the SharpmakeGen that's in the sample directory. So you could in your fork just change the references to DevEnv.vs2015 for DevEnv.vs2017.
What's coming is just that the sln and most csproj will not be under version control, and you'll have to run a bootstrap script to get them.

Just to make sure, did you mean even when opening the solution in vs2017 the editorconfig is ignored?

Reply:
I mean that editorconfig is not natively supported in 2015 and since the projects are generated as 2015 the editorconfig won't have any effect unless you install certain extensions or manually open in 2017.